worldengine/world.py

```diff
@@ -351,7 +350,7 @@ def sea_level(self):
     # Tiles around
     #
 
-    def on_tiles_around_factor(self, factor, pos, radius=1, action=None):
+    def on_tiles_around_factor(self, factor, pos, action, radius=1):
```

Comment:
ok, have you checked all the invokation of this method, given we are changing the order of the parameters? Same goes for the following function